security/fma-l1-pectra.md

```diff
@@ -0,0 +1,95 @@
+# Upgrade 12 (L1 Pectra Defense): Failure Modes and Recovery Path Analysis
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Introduction](#introduction)
+- [Generic Items](#generic-items)
+- [Audit Requirements](#audit-requirements)
+- [Action Items](#action-items)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+|                     |                   |
+| ------------------- | ----------------- |
+| Author              | George Knee       |
+| Created at          | 5th February 2025 |
+| Needs Approval From |                   |
+| Other Reviewers     |                   |
+| Status              | In Review         |
+
+## Introduction
+
+This document is intended to be shared in a public space for reviews and visibility purposes. It covers Upgrade 12, which involves the following changes:
+
+- (Consensus Layer) Ability to parse and validate blocks and receipts with EIP-7702 transactions, as well as blocks with a non-nil EIP-7685 requestsHash field.
+- (Smart Contracts) For chains using Fault Proofs, L1 contracts updates which reference an updated absolute prestate hash
+
+Each change has its own section below with a list of Failure Modes.
+
+Below are references for this project:
+
+- [Github tracker](https://github.com/orgs/ethereum-optimism/projects/117/views/9)
```

Comment:
This view does not exist.

Reply:
yes would be nice to fix it with the correct link.

Reply:
Yeah it got deleted unfortunately. I will need to rebuild it again :-(

Reply:
Updated it here [f3c7d1c](https://github.com/ethereum-optimism/design-docs/pull/197/commits/f3c7d1cff9467429158c13d9771a5ab462c136aa)